source/Engine.cpp

```diff
@@ -27,21 +29,7 @@ bool Engine::Init()
 
 void Engine::Update()
 {
-	myJobSystem.AddJob(job);
-	myJobSystem.AddJob(job);
-	myJobSystem.AddJob(job);
-	myJobSystem.AddJob(job);
-	myJobSystem.AddJob(job);
-	myJobSystem.AddJob(job);
-	myJobSystem.AddJob(job);
-	myJobSystem.AddJob(job);	
-	myJobSystem.AddJob(job);
-	myJobSystem.AddJob(job);
-	myJobSystem.AddJob(job);
-	myJobSystem.AddJob(job);
-	myJobSystem.Terminate();
-
-	myEntityComponentSystem.CreateEntity("New entity 1");
```

Comment:
Is it necessary to keep submitting test code to branches? Made for sub-branches, definitely not main. Think about it. (Can clutter the changes you are actually making. Atleast in my opinion.)

Reply:
I think test code should be removed in pretty much all classes except for Engine.cpp and main, as their purpose during development is to run test code, although it can be argued that test cases could break someone's work (which is undesirable). I'll remove it for now.